dot/sync/chain_processor_integration_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TODO: add test against latest gssmr runtime
```

Comment:
Do we have issues created for these new tests? I am doing a few in core/services but not the whole codebase